gapis/resolve/dependencygraph2/builder.go

```diff
@@ -0,0 +1,303 @@
+// Copyright (C) 2018 Google Inc.
```

Comment:
Maybe we can rename this file to `dependency_graph_builder.go`, as we have two different builders in this directory.

Reply:
Done